docs/reference/hooks.rst

```diff
@@ -1351,6 +1350,81 @@ Hooks for working with registered Snippets.
     def remove_snippet_listing_button_item(buttons, snippet, user, context=None):
         buttons.pop()  # Removes the 'delete' button
 
+
+Bulk actions
+------------
+
+Hooks for registering and customising bulk actions
+
+
+.. _register_bulk_action:
+
+``register_bulk_action``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Registers a new bulk action class to add to the list of bulk actions in the explorer
+
+  This hook can be added to a sub-class of ``BulkAction`` . For example:
```

Comment:
suggestion: this needs rephrasing a bit. Maybe "This hook must be registered with a sub-class of ``BulkAction``."?